hebiros/src/hebiros_group_registry.cpp

```diff
@@ -0,0 +1,39 @@
+#include "hebiros_group_registry.h"
+
+namespace hebiros {
+
+  // TODO: temporary! remove!
+  HebirosGroupRegistry& HebirosGroupRegistry::Instance()
+  {
+    static HebirosGroupRegistry s;
+    return s;
+  }
+
+  HebirosGroupRegistry::HebirosGroupRegistry() {
+  }
+
+  void HebirosGroupRegistry::addGroup(const std::string& name, std::unique_ptr<HebirosGroup> group) {
+    _groups[name] = std::move(group);
```

Comment:
[Transparent comparators](https://stackoverflow.com/questions/20317413/what-are-transparent-comparators) here would be great 🧐